### ISSUE TITLE
Changing theme either does nothing or crashes

### ISSUE THREAD

**Issue by OfekShilon** [rstudio/rstudio#10693]
(Seems new in RStudio 2022.02.0.443)
I'm trying to select themes from Tools/Global options/Appearence/Rstudio theme, and it has no effect.
Attached is a capture from a windows RStudio desktop build. The same seen on Ubuntu 20.
On rstudio-server over ubuntu, there was an additional 'classic' theme in the dropdown, and changing to it *crashed the server* - in fact I wasn't able to restart it so I can't attach logs (maybe the theme persisted in some config file and repeatedly crashes it?)

Hopefully it is reproducible for you.

https://user-images.githubusercontent.com/73080/156004352-d107c3db-bbcd-43b2-9103-26af4c2d4371.mp4

### System details

    RStudio Edition : Seen on Desktop + Server
    RStudio Version :   2022.02.0 build 443
    OS Version      :  seen on Windows 11 + Ubuntu 20
    R Version       :  seen on 4.0.5, 4.1.2

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@OfekShilon Thank you for raising this! For RStudio Server when you say that the server crashes, do you mean that the individual session gets disconnected, or do you mean that the entire server crashes? What I'm seeing (on Ubuntu 20.04, upgrading from Ghost Orchid to Prairie Trillium) is that the individual user's session is disconnected and restarted, with the error below in the logs, but the server continues to run.

```
rsession-rstudio[684996]: WARNING Invalid values found in /home/rstudio/.config/rstudio/rstudio-prefs.json: /global_theme; LOGGED FROM: rstudio::core::Error rstudio::session::prefs::PrefLayer::loadPrefsFromFile(const rstudio::core::FilePath&, const rstudio::core::FilePath&) src/cpp/session/prefs/PrefLayer.cpp:165
```

**Comment by OfekShilon** [comment]
@ronblum  sorry - you are right, the server seems to still be running, but I can no longer start a session on it. Even after force-killing and restarting the server.

**Comment by ronblum** [comment]
@GCRev Correct. The way I tried it was to set the theme to "Classic" in Ghost Orchid, then upgrade to Prairie Trillium.

**Comment by GCRev** [comment]
I also saw that prefs warning after performing the upgrade from GO with "Classic" theme to PT.

I gave that approach a shot and I still can't repro the unresponsive-server part of this. @mikebessuille can we get one or two other devs to see if they can reproduce this at some level?

**Comment by mikebessuille** [comment]
I think a key next step is to provide a workaround to try, to clear the classic setting from the config file or clear the user session data.

**Comment by ronblum** [comment]
Does resetting the state using the instructions at https://support.rstudio.com/hc/en-us/articles/218730228-Resetting-a-user-s-state-on-RStudio-Workbench-RStudio-Server resolve the issue?

**Comment by OfekShilon** [comment]
@ronblum I'm afraid I can no longer check - we reverted the server installation to a previous version

**Comment by OfekShilon** [comment]
@ronblum  I believe I had a session open during the upgrade - can't say if with or without a browser tab open on it.
I can say that the previous version (that we reverted to) is 1.4.1103.
Also, I'm not sure I *left* the theme on "classic". More likely I fiddled with all 3 of them, clicked "apply" and moved on (saw no impact, and no message requiring a restart).  I mentioned "classic" specifically because afterwards, while trying to reproduce on other platforms, I saw it was missing there.

**Comment by OfekShilon** [comment]
@ronblum  That's a reasonable suspicion. I can guarantee that it was there even though (as you say) it wasn't supposed to be.

**Comment by ronblum** [comment]
Did one more check: Upgrading from RStudio Server 1.4.1103 instead of 2021.09.2+382.

- User logged out: Classic theme was not an option, and didn't cause a problem.
- User logged in: Classic theme was still an option, since the browser still had the previous IDE (until logged out, then back in) but did not cause a problem.

So, I'm still unable to reproduce the issue.

**Comment by mikebessuille** [comment]
Closing as not reproduced.
